src/kirby/components/list/list-cell/list-cell.component.tns.scss

```diff
@@ -0,0 +1,16 @@
+.cell {
+    padding: 0 8 0 8;
```

Comment:
Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?